.github/workflows/ci-go-cover.yml

```diff
@@ -25,9 +25,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
-    - name: Install test cases
-      run: make install
+      uses: actions/checkout@v2      
```

Comment:
should we upgrade to `actions/checkout@v3`?

Reply:
Sure